programming_challenges/hacker_rank/update-list-anonymous.clj

```diff
@@ -0,0 +1,14 @@
+;; https://www.hackerrank.com/challenges/fp-update-list/problem
+
```

Comment:
Avoid to use anonymous fn, it's better using as below, when you made aux fns to improve readability.